### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by karimmtarek** [montrealrb/Montreal.rb#168]
I'm kind of curious about the status of the project, like:
- Who is maintaining it?
- How long it is going to take to review a PR?

I'm asking because I've a PR waiting to get reviewed for few days now, and I'm really eager to get it merged, so I can move on to the next story/task. 

I know that all of us have day jobs and time is really in a short supply :) I just want to know so I can manage my own expectations.

**Comment by benichu** [comment]
@karimmtarek Sorry about that, and thank you for starting the discussion.
As I mentioned to @cawel, @sophiedeziel, @mathieugagne and @nicholasjhenry last tuesday during the meetup, we are fully aware of this, and are currently working on a clear plan for both montreal.rb as an organization and any underlying projects such as the website...
I will share a preliminary plan with all of you before the end of the week, that will, I hope, manage your expectations.
Thanks for your great work, stay tuned...

**Comment by karimmtarek** [comment]
@benichu thanks so much for getting back :+1: 
Please feel free to close this, or leave it open in case someone else wants to discuss, it's your call :)

**Comment by benichu** [comment]
I will definitely leave this open until I come back, as promised, with a complete response.

**Comment by benichu** [comment]
I also discussed this with @mdeloupy last Tuesday :smile:

**Comment by karimmtarek** [comment]
Although I was in the meetup Tuesday it looks like I missed some really interesting discussions :cry:

**Comment by benichu** [comment]
How did I miss you!??? Next time come up to me, If I don't...

**Comment by karimmtarek** [comment]
@benichu next time I will :+1:

**Comment by steakunderscore** [comment]
Just bumping the above question. What's the state of this project? I'm happy to do some work on it over the next couple of weeks, but would need some guidance.

**Comment by mathieugagne** [comment]
Christmas time being what it is, it seems we haven't renewed our efforts as much in the new year. The project is definitely on and the organization is strong.
I'll do my best to give the necessary guidance. @benichu @cawel @mdeloupy @garyharan @sophiedeziel @nicholasjhenry Please jump in whenever you think I'm delusional.

@steakunderscore I'm keen to have a quick hangout if you'd think it necessary.

**Comment by steakunderscore** [comment]
Is anyone keen to get things project moving again? Can I get some code reviewed?

**Comment by mathieugagne** [comment]
I've done a few this morning and will carry on later today. Sorry about the delay. We all really appreciate your work and hope to keep the ball rolling for you

**Comment by sophiedeziel** [comment]
Thank you @mathieugagne for what you have done this week. 

I will spend some time tonight as well. I hope I can work on it this weekend too!

**Comment by mathieugagne** [comment]
Thanks to @steakunderscore ! I'm just the supervisor on the side looking at the real working guy, nodding my head in approval at what he does. But thanks :)

**Comment by sophiedeziel** [comment]
I've been pretty active in the past week. Here is a summary of what I have done so far:

- I reviewed and merged the "easiest" pull-requests to handle
- I shut down the old wordpress site and put this one into production!! :tada:
- I cleaned up opened issues that have been solved by merged pull-request (thanks to @steakunderscore for pointing some to me)
- I created a `priority` label to help focus the work on specific aspects. Right now, there are around 10 issues targeting Events. Those are the features that will be the most useful in the near future. When they are done, I will set the priority to the job postings. 
- I opened some pull requests that should be reviewed. I don't think they require a lot of work. If someone is available (@benichu @cawel @mdeloupy @nicholasjhenry) to review them, I have a lot of emojis to express my gratitude.
